utils/fo-installdeps

```diff
@@ -209,9 +209,9 @@ if [[ $RUNTIME ]]; then
          # enable, possible init, and start postgresql
          /sbin/chkconfig postgresql on
          if [ ! -f /var/lib/pgsql/data/PG_VERSION ]; then
-            /sbin/service postgresql initdb
+            /sbin/service postgresql --initdb
```

Comment:
This command failed for me on Fedora 27 and CentOS 7
```suggestion
            /usr/bin/postgresql-setup --initdb
```

Reply:
I have made the changes, please check it now.